pkg/plugins/runtime/k8s/webhooks/injector/precheck.go

```diff
@@ -127,3 +128,41 @@ func (i *KumaInjector) isInjectionException(pod *kube_core.Pod) bool {
 	}
 	return false
 }
+
+// these are all existing annotations that are supported by metadata.GetBooleanWithDefault
+var booleanAnnotations = map[string]bool{
+	metadata.KumaTrafficDropInvalidPackets:         true,
+	metadata.KumaTrafficIptablesLogs:               true,
+	metadata.KumaWaitForDataplaneReady:             true,
+	metadata.KumaTransparentProxyingEbpf:           true,
+	metadata.KumaBuiltinDNS:                        true,
+	metadata.KumaBuiltinDNSLogging:                 true,
+	metadata.KumaGatewayAnnotation:                 true,
+	metadata.KumaSidecarInjectionAnnotation:        true,
+	metadata.KumaIngressAnnotation:                 true,
+	metadata.KumaEgressAnnotation:                  true,
+	metadata.KumaSidecarInjectedAnnotation:         true,
+	metadata.KumaTransparentProxyingAnnotation:     true,
+	metadata.KumaMetricsPrometheusAggregateEnabled: true,
+	metadata.KumaVirtualProbesAnnotation:           true,
+	metadata.KumaIgnoreAnnotation:                  true,
+	metadata.KumaInitFirst:                         true,
+}
+
+func logYesNoDeprecations(podAnnotations map[string]string, logger logr.Logger) bool {
+	for key, value := range podAnnotations {
+		if _, isBooleanAnno := booleanAnnotations[key]; !isBooleanAnno {
+			continue
```

Comment:
Should we log here if ever we forget to update the list above?

Reply:
Umm, you mean if we introduce new boolean annotations before we remove this deprecation?

Reply:
I think it's OK to leave it, we may delete this deprecation very soon in the next release.